Public/Src/Cache/ContentStore/UtilitiesCore/FileSystemConstants.cs

```diff
@@ -12,6 +13,26 @@ namespace BuildXL.Cache.ContentStore.Interfaces.FileSystem
     /// </summary>
     public static class FileSystemConstants
     {
+        private static class LibC
+        {
+            /// <nodoc />
+            [DllImport("libc", SetLastError = true)]
+            public static extern int chmod(string path, int mode);
+        }
+
+        /// <nodoc />
+        public static void OverrideFileAccessMode(bool changePermissions, string path)

```

Comment:
>OverrideFileAccessMode [](start = 27, length = 22)

This utility doesn't seem to fit with the purpose of the FileSystemContants class. Is there a better home for it?

Reply:
I've added a `UnixHelpers` class, which will make it easier to delete once we have proper support implemented in cache.